src/main/views/daily-cause-list.njk

```diff
@@ -66,7 +66,11 @@
 
             {% set judiciaryJudges = '' %}
             {% for judge in session['judiciary'] %}
-              {% set judiciaryJudges = judiciaryJudges + ', ' + judge['johTitle'] + ' ' + judge['johNameSurname'] %}
+              {% if judge['johTitle'] | length %}
+                {% set judiciaryJudges = judiciaryJudges + ', ' + judge['johTitle'] + ' ' + judge['johNameSurname'] %}
```

Comment:
I believe we should use johKnownAs for this instead of the names. Would you be able to double check?